internal/config/main_config.go

```diff
@@ -67,7 +72,8 @@ var Logging = &config.Logging
 var Cloudwatch = &config.Cloudwatch
 var AWS = &config.AWS
 var Features = &config.FeatureFlags
-var ImageBuilderURL = &config.ImageBuilder.URL
+var ImageBuilderURL = &config.RestEndpoints.ImageBuilder.URL
+var SourcesURL = &config.RestEndpoints.Sources.URL
```

Comment:
I would only export RestEndpoints here to follow the pattern, but it's okay.

    var RestEndpoints = &config.RestEndpoints